src/buffer/out/OutputCellView.hpp

```diff
@@ -6,7 +6,7 @@ Module Name:
 - OutputCellView.hpp
 
 Abstract:
-- Read-only view into a single cell of data that someone is attempting to write into the output buffer.
+- Read view into a single cell of data that someone is attempting to write into the output buffer.

```

Comment:
It's.. not read-only any more? You can write through it?

Reply:
I mean, you can now UpdateSomething in it, which disqualified the “read-only” part, right?

Reply:
Also I've tried various ways to keep the original `_GenerateView` (and preserve the read-only-ness in `OutputCellView`), but I failed to find a way to achieve the same level of performance as the current implementation. I've discussed with @lhecker about this, and he seems to concur with the solution.